sql/src/test/java/io/druid/sql/calcite/CalciteQueryTest.java

```diff
@@ -449,10 +509,37 @@ public void testSelectStar() throws Exception
   }
 
   @Test
+  public void testSelectStarOnForbiddenTable() throws Exception
+  {
+    assertQueryIsForbidden(
```

Comment:
Looks better to use ExpectedException.

Reply:
I was thinking of doing that initially, but saw there were similar `assertQueryIsUnplannable` methods in the test. This also lets me put both the query as unauthorized user and query as authorized superuser in the same test, I decided to keep the assertQueryIsForbidden method, it could be useful for future tests that try multiple queries where access is denied.

Reply:
Ah I see. Then please leave some notes.

Reply:
Added a comment to that method